src/pages/workspace/accounting/netsuite/advanced/NetSuiteAdvancedPage.tsx

```diff
@@ -0,0 +1,286 @@
+        if (config?.approvalAccount === CONST.NETSUITE_APPROVAL_ACCOUNT_DEFAULT) {
+            return {
+                id: CONST.NETSUITE_APPROVAL_ACCOUNT_DEFAULT,
+                name: translate('workspace.netsuite.advancedConfig.defaultApprovalAccount'),
+            };
+        }
+        return (payableList ?? []).find((payableAccount) => payableAccount.id === config?.approvalAccount);
+    }, [config?.approvalAccount, payableList, translate]);
+
+    const menuItems: Array<MenuItem | ToggleItem | DividerLineItem> = [
+        {
+            type: 'toggle',
+            title: translate('workspace.accounting.autoSync'),
+            subtitle: translate('workspace.netsuite.advancedConfig.autoSyncDescription'),
+            isActive: !!autoSyncConfig?.autoSync.enabled,
+            switchAccessibilityLabel: translate('workspace.netsuite.advancedConfig.autoSyncDescription'),
+            shouldPlaceSubtitleBelowSwitch: true,
+            onCloseError: () => Policy.clearNetSuiteAutoSyncErrorField(policyID),
+            onToggle: () => Connections.updateNetSuiteAutoSync(policyID, !autoSyncConfig?.autoSync.enabled, !!autoSyncConfig?.autoSync.enabled),
+            pendingAction: autoSyncConfig?.pendingFields?.autoSync,
+            errors: ErrorUtils.getLatestErrorField(autoSyncConfig, CONST.NETSUITE_CONFIG.AUTO_SYNC),
+        },
+        {
+            type: 'divider',
+            key: 'divider1',
+        },
+        {
+            type: 'toggle',
+            title: translate('workspace.accounting.reimbursedReports'),
+            subtitle: translate('workspace.netsuite.advancedConfig.reimbursedReportsDescription'),
+            isActive: !!config?.syncOptions.syncReimbursedReports,
+            switchAccessibilityLabel: translate('workspace.netsuite.advancedConfig.reimbursedReportsDescription'),
+            shouldPlaceSubtitleBelowSwitch: true,
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.SYNC_OPTIONS.SYNC_REIMBURSED_REPORTS),
+            onToggle: () => Connections.updateNetSuiteSyncReimbursedReports(policyID, !config?.syncOptions.syncReimbursedReports, !!config?.syncOptions.syncReimbursedReports),
+            pendingAction: config?.syncOptions.pendingFields?.syncReimbursedReports,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.SYNC_OPTIONS.SYNC_REIMBURSED_REPORTS),
+            shouldHide: config?.reimbursableExpensesExportDestination === CONST.NETSUITE_EXPORT_DESTINATION.JOURNAL_ENTRY,
+        },
+        {
+            type: 'menuitem',
+            description: translate('workspace.netsuite.advancedConfig.reimbursementsAccount'),
+            onPress: () => {}, // TODO: This will be implemented in a future PR
+            brickRoadIndicator: config?.errorFields?.reimbursementAccountID ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: selectedReimbursementAccount ? selectedReimbursementAccount.name : undefined,
```

Comment:
I think we discussed this in the thread earlier, if no value is selected, we pick the first one in the list.